homeassistant/components/growatt_server/sensor.py

```diff
@@ -32,7 +32,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SCAN_INTERVAL = datetime.timedelta(minutes=1)
+SCAN_INTERVAL = datetime.timedelta(minutes=5)
```

Comment:
Note that this determines the interval between each probe's api calls. It's not a timeout as written in the PR description.

Reply:
Thanks I was in a hurry this morning to get this done before I went to work. I've updated the description to say "Polling interval" instead.